config/set/datetime-to-carbon.php

```diff
@@ -15,4 +16,11 @@
         DateTimeMethodCallToCarbonRector::class,
         TimeFuncCallToCarbonRector::class,
     ]);
+
+    // Replace instaces where first arg is not a string
+    $rectorConfig
+        ->ruleWithConfiguration(RenameClassRector::class, [
+            'DateTime' => 'Carbon\Carbon',
+            'DateTimeImmutable' => 'Carbon\CarbonImmutable',
+        ]);
```

Comment:
please revert this, as this will bc break, as it will also any `DateTime` reference:

```php
function foo(DateTime $d)
{
}
```

see https://getrector.com/demo/47c44de7-64ef-4645-9808-0c46515c3bb6